Makefile

```diff
@@ -154,9 +154,11 @@ ifneq ($(PR), false)
 endif
 
 initdb:
+	docker-compose run --rm web psql -h db -d postgres -U postgres -c "SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname ='warehouse';"
```

Comment:
disconnects any other sessions to make `make initdb` run regardless of if `make serve` containers are accessing the db.